packages/xo-remote-parser/src/index.js

```diff
@@ -15,6 +18,17 @@ const parseOptionList = (optionList = '') => {
     optionList = optionList.substring(1)
   }
   const parsed = queryString.parse(optionList)
+
+  // bugfix for persisting error notification
+  if ('error' in parsed) {
+    warn('Deleting "error" value in url query, resave your remote to clear this values', { error: parsed.error })
```

Comment:
These logs are not (easily) visible by users, they only appear in xo-server's system logs.

Do not use them if you want users to act on them.